protos/test_result.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wptdashboard;
+
+import "bq_table_name.proto";
+import "browser.proto";
+import "engine.proto";
+import "google/protobuf/timestamp.proto";
+import "operating_system.proto";
+import "sub_test_status.proto";
+import "test_status.proto";
+
+// An result for a single test or sub-test from a TestRun.
+message TestResult {
+  option (gen_bq_schema.table_name) = "test_results";
+
+  OperatingSystem os = 1;
+  string os_version_str = 2;
+
+  Engine engine = 3;
+  Browser browser = 4;
+  string browser_version_str = 5;
+
+  google.protobuf.Timestamp test_run_start_time = 6;
+  google.protobuf.Timestamp test_run_end_time = 7;
+  string long_wpt_hash = 8;
+  google.protobuf.Timestamp wpt_commit_time = 9;
```

Comment:
The `wpt` data should both be `wpt_*`, ie. `wpt_hash` & `wpt_commit_time` (although, I'm not sure that `wpt_commit_time` is even necessary)

Reply:
SGTM. I'll add a comment that it's the full-length hash. The commit_time is intended to ease BigQuery queries concerned with commit time (without having to cross-reference to a checkout of WPT for every row).